src/renderer/components/wallet/NewMnemonicGenerate.tsx

```diff
@@ -40,10 +53,18 @@ const NewMnemonicGenerate: React.FC<Props> = (_: Props): JSX.Element => {
     ],
     []
   )
+
+  const copyPhraseToClipborad = useCallback(() => {
+    navigator.clipboard.writeText(phrase)
+  }, [phrase])
   return (
     <>
-      <h1>Generate the new mnemonic wallet</h1>
+      <Label onClick={copyPhraseToClipborad}>{intl.formatMessage({ id: 'wallet.create.copy.phrase' })}</Label>
+      <MnemonicPhrase words={phraseWords} />
```

Comment:
[suggestion] I would recommend to add a "refresh" button to re-generate the phrase if the user does not agree with it. I've updated the layout at FIGMA  https://www.figma.com/file/LgRqZCGMShZ2mJ702MIqyV/ASGARDEX-2020?node-id=1310%3A0